tensorflow/inference/docker/2.8/py3/cu112/Dockerfile.gpu

```diff
@@ -140,6 +140,7 @@ RUN ${PIP} install -U --no-cache-dir \
     "awscli<2" \
```

Comment:
Can we change [this](https://github.com/aws/deep-learning-containers/pull/2538/files#diff-256c9f375cceea45f3bf8d650a757d33b24004fafe83a0dfc9bcf52a37acee30R152) to tensorflow-serving-api-gpu=="2.8.4"?